### PR TITLE
Bumped Swift version to 4.1.

### PR DESCRIPTION
**Pull request by jacobvanorder**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+osx_image: xcode9.3
 language: swift
 # cache: cocoapods
 # podfile: Example/Podfile
 # before_install:
 # - gem install cocoapods # Since Travis is not always on latest version
 # - pod install --project-directory=Example
 script:
-- set -o pipefail && xcodebuild test -enableCodeCoverage YES -workspace Example/VIPERBuilder.xcworkspace -scheme VIPERBuilder-Example -sdk iphonesimulator11.2 -destination 'platform=iOS Simulator,name=iPhone X,OS=11.2' ONLY_ACTIVE_ARCH=NO | xcpretty
+- set -o pipefail && xcodebuild test -enableCodeCoverage YES -workspace Example/VIPERBuilder.xcworkspace -scheme VIPERBuilder-Example -sdk iphonesimulator11.4 -destination 'platform=iOS Simulator,name=iPhone X,OS=11.4' ONLY_ACTIVE_ARCH=NO | xcpretty
```
Comment (macdoum1) [inline .travis.yml]: I'm seeing a build failure on travis: `xcodebuild: error: SDK "iphonesimulator11.4" cannot be located.`

I believe this needs to be `OS=11.3` & `iphonesimulator11.3`
https://docs.travis-ci.com/user/reference/osx/#Xcode-9.3

Comment (jacobvanorder) [reply .travis.yml]: Yep! It ran locally but wasn't sure of the travis limitations. Thanks!

**Comment by jacobvanorder** [comment]
@macdoum1 quick! approve it before Travis realizes *nothing* changed between failure and pass.

**Review by macdoum1 — APPROVED** [review]
Yay 👍

**Pull request merged**